### ISSUE TITLE
Cannot determine the packages folder to restore NuGet packages.

### ISSUE THREAD

**Issue by AnthonySteele** [NuKeeperDotNet/NuKeeper#72]
On a VS2015 project,  [this line](https://github.com/NuKeeperDotNet/NuKeeper/blob/master/NuKeeper/NuGet/Process/PackagesConfigUpdater.cs#L24) gives an error message: `Cannot determine the packages folder to restore NuGet packages. Please specify either -PackagesDirectory or -SolutionDirectory.`

It works if you run `nuget restore` in the same folder as the `.sln` file instead. Doing this once may be a better option.

**Comment by AnthonySteele** [comment]
It's still an open question if this is in fact an issue with the affected projects.

See https://docs.microsoft.com/en-us/nuget/consume-packages/package-restore#automatic-restore-in-visual-studio

**Comment by AnthonySteele** [comment]
Still an issue with some of our repos. We get:
````
Cannot determine the packages folder to restore NuGet packages.
 Please specify either -PackagesDirectory or -SolutionDirectory.
Unable to locate the packages folder. 
Verify all the packages are restored before running 'nuget.exe update'.
````

a `nuget restore` in the same folder as the solution file does the trick.

**Comment by AnthonySteele** [comment]
Finally fixed by #135
